framework/Optimizers/Conjugate.py

```diff
@@ -49,11 +48,11 @@ class ConjugateGradient(SPSA):
   ##########################
   # Initialization Methods #
   ##########################
-  def __init__(self):
-    """
-      Default Constructor
-    """
-    SPSA.__init__(self)
+  # def __init__(self):
+  #   """
+  #     Default Constructor
+  #   """
+  #   SPSA.__init__(self)
```

Comment:
Just remove this commented code.

Reply:
I do not have those lines already, not sure why it did not show.